atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,6 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        # currently we only support v1+v2 schema1
```

Comment:
I don't want the manifest_digests, and rewriting that code to determine whether we have v2schema1 manifest_digests or v2schema2 manifest_digests should be outside the scope of this task. I just want to know if we've run pulp_push and pulp_pull should return ['application/json'], or if we've run pulp_sync and pulp_pull should return ['application/json', 'application/vnd.docker.distribution.manifest.v1+json']

is there a way for the orchestrator build to know what plugins the worker builds ran?